arm-redis/2015-08-01/swagger/redis.json

```diff
@@ -2,7 +2,7 @@
   "swagger": "2.0",
   "info": {
     "title": "RedisManagementClient",
-    "description": ".Net client wrapper for the REST API for Azure Redis Cache Management Service",
+    "description": ".Net client wrapper for the REST API for Azure Redis Cache Management Service.",
```

Comment:
@steved0x - Dont you think the description should not have keywords like .NET and client in it? The spec is a representation of the API and would be used to generate clients in different languages. Plus it wouldnt look good to have the word client specified in the ref docs.